electron/main.js

```diff
@@ -76,7 +76,8 @@ const handleBitcoinLink = input => {
 const handleLightningLink = input => {
   const address = input.split(':')[1]
   zap.sendMessage('lightningPaymentUri', { address })
-  mainWindow.show()
+  // Hide the window. The app will bring itself into focus if/when needed.
+  mainWindow.hide()
```

Comment:
This will also alter regular, non autopay behavior while `bitcoin:` links handling stays the same

Reply:
That's intended. There should be no change to `bitcoin` links. But for `lightning` links we want to let the app decide wether or not to focus the window based on wether its autopay or not, which happens in the `lightningPaymentUri` method.